econml/metalearners.py

```diff
@@ -262,6 +262,8 @@ def fit(self, Y, T, X, *, inference=None):
         """
         # Check inputs
         Y, T, X, _ = check_inputs(Y, T, X, multi_output_T=False)
```

Comment:
Is check_inputs equivalent to check_input_arrays?  Would it be beneficial to switch things to the latter everywhere for consistency?